src/test/test_arch.cc

```diff
@@ -33,7 +34,7 @@ TEST(Arch, all)
 #if (__arm__ || __aarch64__ || __x86_64__) && __linux__
   char flags[FLAGS_SIZE];
   FILE *f = popen("grep '^\\(flags\\|Features\\)[	 ]*:' "
-                  "/proc/cpuinfo | head -1", "r");
+                  PROCPREFIX "/proc/cpuinfo | head -1", "r");
```

Comment:
@tchaikov 
Have taken this part out of the PR, looked more deeply into it.
And I needed I'll have to fix it in the future.